appservice/src/getKuduClient.ts

```diff
@@ -8,10 +8,15 @@ import { BasicAuthenticationCredentials } from 'ms-rest';
 import { addExtensionUserAgent } from 'vscode-azureextensionui';
 import KuduClient from 'vscode-azurekudu';
 import { ArgumentError } from './errors';
+import { localize } from './localize';
 import { SiteClient } from './SiteClient';
 
 export async function getKuduClient(client: SiteClient): Promise<KuduClient> {
     const user: User = await client.getWebAppPublishCredential();
```

Comment:
Move the error above `await client.getWebAppPublishCredential();`. No need to wait for that operation